src/main/java/org/fundacionjala/coding/carmen/SortContent.java

```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.coding.carmen;
+
+import java.util.Arrays;
+
+final class SortContent {
+
+    private SortContent() {    }
+
+    public static String sortTheInnerContent(final String sentenceInput) {
+        String[] sentence = sentenceInput.split(" ");
+        for (int i = 0; i < sentence.length; i++) {
+            int wordSize = sentence[i].length();
+            String innerString = sentence[i].substring(1, wordSize - 1);
+            String firstChar = sentence[i].substring(0, 1);
+            String lastChar = sentence[i].substring(wordSize - 1, wordSize);
+
+            char[] arr = innerString.toCharArray();
+            Arrays.sort(arr);
+            innerString = new StringBuilder(String.valueOf(arr)).reverse().toString();
+
+            sentence[i] = firstChar + innerString + lastChar;
```

Comment:
You can take advantage of Arrays.sort to sort descending 
```java
            String[] innerArray = innerString.split("");
            Arrays.sort(innerArray, Comparator.reverseOrder());
            sentence[i] = firstChar + String.join("", innerArray) + lastChar;
```

Reply:
thanks, observation resolved